docs/installation/generic.md

```diff
@@ -153,11 +170,18 @@ export REDIS_URL=redis://localhost:6379/0
 ```bash
 celery worker -A app.celery &
 ```
-
+### Note: The '&' in the above command means detaching from the console. To avoid use the following command instead
+```bash
+celery worker -A app.celery --loglevel=INFO
+```
 ##### Starting the app generator web server
 ```bash
 gunicorn -b 0.0.0.0:8080 app:app --enable-stdio-inheritance --log-level "info"
 ```
 
 You can now visit [http://localhost:8080](http://localhost:8080/) on your browser to access the app generator.
 
+### A FEW PRECAUTIONS/TIPS:
```

Comment:
https://github.com/mahikaw/open-event-android/blob/3702b9f311d60e6def42e99636160e2267bca6eb/docs/installation/generic.md#install-brew

Also here font size more , please change it along with it.
Just view preview and do appropriate changes so that doc looks good :smile: :+1: